### PR TITLE
Integration+linaro warp7 tbb

### PR DESCRIPTION
**Pull request by bryanodonoghue**
Switch on Trusted Boot for the WaRP7.

- Describes a FIP layout with necessary key material
- Subtracts the Kernel DTB from the FIP in-line with MBED Linux boot flow for WaRP7
- Updates documentation to describe the updated flow

### CODE REVIEW

**Comment by bryanodonoghue** [comment]
@niej

File: plat/imx/imx7/warp7/warp7_trusted_boot.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <platform.h>
```
Comment (antonio-nino-diaz-arm) [inline plat/imx/imx7/warp7/warp7_trusted_boot.c]: Use the full path.

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

File: plat/imx/imx7/warp7/warp7_trusted_boot.c (hunk 1 above)
Comment (bryanodonoghue) [reply plat/imx/imx7/warp7/warp7_trusted_boot.c]: done

**Comment by antonio-nino-diaz-arm** [comment]
jenkins: test this please

**Comment by bryanodonoghue** [comment]
Hmm.
I can't view the output of that build failure - is the server down ?

**Comment by antonio-nino-diaz-arm** [comment]
Sorry, you can't access that server. I believe it wasn't an issue caused by your patch but something else and I forgot to mention it here.

**Comment by antonio-nino-diaz-arm** [comment]
Do we need to wait for @niej to review the changes?

**Comment by niej** [comment]
Acked-by: Jun Nie <jun.nie@linaro.org>

**Pull request merged**